webapp/src/containers/components/modals/actionModal.tsx

```diff
@@ -5,21 +5,28 @@ import Table from 'react-bootstrap/Table';
 import Form from 'react-bootstrap/Form';
 import './styles.css';
 
+// eslint-disable-next-line import/no-unresolved
+import {Configs} from 'types/plugin/common';
+
 interface Props {
```

Comment:
it should by a `type`
learn a bit about the differences and uses of `interface` and `type` in TS.